test/model_tests/location_test.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:beacon/models/location/location.dart';
+import 'package:test/test.dart';
+
+void main() {
+  Map<String, dynamic> dummyJson = {"lat": "100.1", "lon": "200.1"};
+
+  test('Checking if location fetch from Json is working: ', () {
+    Location location = Location.fromJson(dummyJson);
```

Comment:
Location model is being used inside beacon model only and PR #114 adds check for location fields too. It would be better if you both collaborate on single PR.

Reply:
Currently there's no use of this fetch method, but in future it may be required for seeing if a user is online keeping in mind issue #4. Should I keep it for that or is it needless?